recipes/_configure_indexes.rb

```diff
@@ -51,8 +52,12 @@
 
     padding ||= default_config['_dataSizePaddingPercent']
     padding = padding.nil? ? 1.1 : 1 + (padding / 100.0)
-
-    hash['maxTotalDataSizeMB'] = (daily_mb * padding * frozen_time_in_days * replication_factor).to_i / indexer_count
+    # For smartstore enabled indexes only .
+    if s2_enabled_index == true
+      hash['maxGlobalDataSizeMB'] = (daily_mb * padding * frozen_time_in_days * replication_factor).to_i / indexer_count
```

Comment:
This calculation should be different since this is a global value instead of per indexer. I don't think the rep factor or indexer count should be relevant here.